app/views/search/index.html.haml

```diff
@@ -56,21 +59,21 @@
                 = t('search.hide_advanced_options')
     - unless @federation_errors.blank?
       = render Display::AlertComponent.new(message: @federation_errors, type: "warning")
-    - if @search_results
```

Comment:
why removing this?

Reply:
it has no sense, we have a loop `@search_results.each` in the next line (also `@search_results `can't be `nil` but `[]` so if I put it back we need to use `unless @search_results.blank?` instead

Reply:
If not needed don't put it back